manageiq-ui-classic.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.executables   = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency "rails", "~> 5.0.0", ">= 5.0.0.1"
+  s.add_dependency "rails", ">= 5.0.0.1", "< 5.1"
```

Comment:
Yeah, with a typical gem, it probably should be looser than the application, but for this case in particular, it is cyclically depended on the main repo in development (we work around it so it isn't in reality, but what I mean is that ManageIQ/manageiq is a dependency for this gem [in development/test envs](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/Gemfile#L17-L24)), and in ManageIQ/manageiq we are basically setting the harder dependency that will eventually win out.

Really, I see this repo as a "external, but tightly coupled component" of `manageiq`, and not a individual standalone entity, at least as it stands right now.  Granted, this may change as this project evolves, but with how we are handling the Gemfile current, I feel like `manageiq` will always be setting the hard rule regardless if which codebase you happen to be working on directly.  I think that leaving the difference between the two projects in place will lead to lots of churn in version changes (and lots of tweets like the one linked above  😉 )

If you do decide that loosening the Rails dependency as the final solution, I would suggest just removing the lower bound on the dependency:

```diff
- s.add_dependency "rails", "~> 5.0.0", ">= 5.0.0.1"
+ s.add_dependency "rails", ">= 5.0.0.1"
```

I still think we should match, but this would have the benefit of requiring you to change this less often.